modules/administration/pages/ptfs.adoc

```diff
@@ -0,0 +1,81 @@
+[[ptfs]]
+= Using PTFs in {productname}
+
+SUSE provide for their Enterprise Products temporary fixes shipped to individual customers.
+These PTFs (Program Temporary Fixes) can now be made available as repositories which can be synced in {productname}.
+
+
+== Understanding PTF packages
+
+PTF packages are installed via a proxy package and are named `ptf-xxxxxx`.
+They will depend on the correct version of the package that is known to include the correction in the software. This type of package
+
+* cannot be installed accidentally (ie. zypper update will never suggest installing them),
+* cannot be removed accidentally (ie. a newer package version will not replace the PTF one, unless the user makes it explicitly on the zypper command line),
+* is only updated when the newer version is known to address the specific issue addressed first by the PTF,
+* updates only packages already installed on the system (a software that is split in multiple packages will have only the packages currently installed on the system being replaced).
```

Comment:
```suggestion
* cannot be installed accidentally (i.e. zypper update will never suggest installing them),
* cannot be removed accidentally (i.e. a newer package version will not replace the PTF one, unless the user makes it explicitly on the zypper command line),
* is only updated when the newer version is known to address the specific issue previously solved by the PTF,
* updates only packages already installed on the system (i.e. if a software is split into multiple packages, the PTF will replace only those currently installed on the system).
```

Reply:
This is a 1:1 copy of the TID. I will let our doc team correct this or not